sig-node/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing on Sig-Node
+
+Welcome!
+
```

Comment:
If we're using @dims's guide https://hackmd.io/@XYdYH0X5SYC3DUYFF5Wylg/SyYoo-U1w as a starting point, there's a lot of missing links we could include in here.

I also recommend we include links to the Node Triage documentation.

Reply:
+1, 
Also add a link to developer docs https://github.com/kubernetes/community/tree/master/contributors/devel/sig-node. It has lots of docs specific to SIG Node.

Reply:
Added more